Lib9c/Action/DailyReward.cs

```diff
@@ -38,13 +38,13 @@ public override IAccountStateDelta Execute(IActionContext context)
                 throw new FailedLoadStateException($"{addressesHex}Aborted as the game config was failed to load.");
             }
 
-            if (avatarState.dailyRewardReceivedIndex < context.BlockIndex - gameConfigState.DailyRewardInterval)
+            if (context.BlockIndex - avatarState.dailyRewardReceivedIndex < gameConfigState.DailyRewardInterval)
             {
                 var sb = new StringBuilder()
                     .Append($"{addressesHex}Not enough block index since the last received daily reward.")
                     .Append(
-                        $" Expected: Equals or greater than {context.BlockIndex - gameConfigState.DailyRewardInterval}.")
-                    .Append($" Actual: {avatarState.dailyRewardReceivedIndex}");
+                        $" Expected: Equals or greater than ({gameConfigState.DailyRewardInterval}).")
```

Comment:
It may be trivial but `context.BlockIndex` should equal or be greater than `avatarState.dailyRewardReceivedIndex + gameConfigState.DailyRewardInterval`, not `gameConfigState.DailyRewardInterval` IMO.